yb-voyager/cmd/root.go

```diff
@@ -66,6 +70,11 @@ Refer to docs (https://docs.yugabyte.com/preview/migrate/) for more details like
 			go startPprofServer()
 		}
 
+		controlPlane = yugabyted.New(exportDir)
+		err := controlPlane.Init()
+		if err != nil {
+			log.Warnf("Failed to initialize the target DB for visualization. %s", err)
+		}
```

Comment:
We should have two env vars:
- CONTROL_PLANE_TYPE=yugabyted
- YUGABYTED_DB_CONN_STRING=postgres://...

If the `CONTROL_PLANE_TYPE` is not set, instantiate a `NoopControlPlane` type.
If the env var is set to `yugabyted`, then
- ensure that YUGABYTED_DB_CONN_STRING is set, and
- instantiate `YugabyteD`.

Reply:
Incoorporated the comments.